.github/workflows/release.yml

```diff
@@ -0,0 +1,38 @@
+name: Release collection
+
+on:
+  push:
+    tags:
+      - "v*.*.*"
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
```

Comment:
@sabre1041 @hcherukuri I suppose we can't run on Fedora?

Reply:
I am not sure,  Need to check or Andrew might know.

Reply:
Neh, forget it. I recall Andy and I already discussed that, I think there is, indeed, no other choice.